src/common/cssHelpers/form.module.css

```diff
@@ -139,7 +139,7 @@
   
```

Comment:
dev.hel.fi recommends _camelCase_ for style file names: https://dev.hel.fi/coding-standards#naming

Reply:
All old profile css files are named *.module.css. Probably because name is required so React identifies them as css modules. 

https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/